phoenix5-spark/README.md

```diff
@@ -293,6 +293,17 @@ to executors as a comma-separated list against the key `phoenixConfigs` i.e (Pho
     Note that the same property values will be used for both the driver and all executors and
     these configurations are used each time a connection is made (both on the driver and executors).
 
+- As of [PHOENIX-7377](https://issues.apache.org/jira/browse/PHOENIX-7377), you can pass boolean parameter to avoid mapping
+  non default family columns to `columnFamily.columnName` by setting the key `escapeColumnFamily` to `true` (default value: `false`), for ex:
```

Comment:
'escapeColumnFamily' is not a descriptive property name for this property.

Something like 'doNotMapColumnFamily' would describe the functionality better.

Reply:
I renamed the property to 'doNotMapColumnFamily'.